Tutorial_1c_Line_Ratios.ipynb

```diff
@@ -0,0 +1,736 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.    cbar.set_label('Integrated Intensity (K km/s)')
</pre><p>Line ratio instead of integrated intensity.</p><p> </p><p>Also note that you can auto set the units from the moment 0 maps when plotting by themselves:</p>

<pre class="ql-syntax" spellcheck="false">cbar.set_label(f'Integrated Intensity ({masked_moment0_12co.unit})')
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Smithsonian/sma-science-tutorials/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>